### PR TITLE
[google-map-react]: add types for 1.1.5

### PR DESCRIPTION
**Pull request by ebidel**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [ ] `tslint.json` should be present and it shouldn't have any additional or disabling of rules. Just content as `{ "extends": "dtslint/dt.json" }`. If for reason the some rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]`  and not for whole package so that the need for disabling can be reviewed.
- [ ] `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/google-map-react/google-map-react/commit/260c6d1c76484a3dbfcb412ae26e98066e1fc520
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Comment by ebidel** [comment]
PTAL @honzabrecka

File: types/google-map-react/index.d.ts [hunk 1]
```diff
@@ -116,36 +116,38 @@ export interface ChangeEventValue {
...
+    onChildMouseUp?(childKey: any, childProps: any, mouse: any): void;
+    onChildMouseMove?(childKey: any, childProps: any, mouse: any): void;
+    onDrag?(map: any): void;
+    onDragEnd?(map: any): void;
+    onZoomAnimationStart?(args: any): void;
+    onZoomAnimationEnd?(args: any): void;
+    onMapTypeIdChange?(args: any): void;
+    distanceToMouse?(pt: Point, mousePos: Point, markerProps?: object): number;
+    googleMapLoader?(bootstrapURLKeys: any): void;
+    onGoogleApiLoaded?(maps: { map: any; maps: any }): void;
```
Comment (androsj) [inline types/google-map-react/index.d.ts]:
```suggestion
    onGoogleApiLoaded?(maps: { map: any; maps: any, ref: Element | null }): void;
```

File: types/google-map-react/index.d.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for google-map-react 1.1.5
+// Type definitions for google-map-react 1.1
```
Comment (androsj) [inline types/google-map-react/index.d.ts]: Was this giving you some kind of error?

Comment (ebidel) [reply types/google-map-react/index.d.ts]: Indeed.

**Review by androsj — APPROVED** [review]
Good to go

**Comment by ebidel** [comment]
How we doing on this?

**Pull request merged**